app/services/flags/reorder_flag.rb

```diff
@@ -22,37 +21,37 @@ class Flags::ReorderFlag
 
   private
 
-  def fetch_flag(flag_id:)
-    Flag.find(flag_id)
+  def fetch_flag(contract:)
+    Flag.find_by(id: contract.flag_id)
   end
 
   def invalid_access(guardian:, flag:)
     guardian.can_reorder_flag?(flag)
   end
 
-  def all_flags
```

Comment:
Using utility methods inside a service is not something we want to encourage. Instead, here, we can simply declare a model.